### PR TITLE
layout

### PR DESCRIPTION
**Pull request by VladVeklenko**
Demo link: https://VladVeklenko.github.io/layout_product-cards/
Test: https://VladVeklenko.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,61 @@
...
+      <p class="container__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="container__productNumber">
+        Product code: 195434
+      </p>
+
+      <div class="container__stars">
+
```
Comment (PavloReutskiy) [inline src/index.html:29]: you don't need this empty line

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  border-radius: 5px;
+  width: 200px;
+  padding: 0 16px;
+}
+
+.container__img {
+  align-self: center;
+  margin-top: 32px;
+  height: 134px;
+  width: 160px;
```
Comment (PavloReutskiy) [inline src/styles/style.css:23]: suggest min-height and max-width instead of height and width

File: src/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  margin-bottom: 40px;
+}
+
+.container__description {
+  margin-bottom: 4px;
```
Comment (PavloReutskiy) [inline src/styles/style.css:28]: you used margin-top on the previous selector 
image

File: src/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap");
+
```
Comment (PavloReutskiy) [inline src/styles/style.css:2]: I would suggest you change the name of the file on card.css or something like that 
style.css can confuse

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,61 @@
...
+      data-qa="card"
+    >
+      <img 
+        src="images/imac.jpeg"
+        alt="Apple A1419 iMac 27 Retina 5K Monoblock "
+        class="container__img"
+      >
+      <p class="container__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
```
Comment (PavloReutskiy) [inline src/index.html:22]: you forgot tag <h2>
it's a good place for it
image

**Review by PavloReutskiy — CHANGES_REQUESTED** [review]
a few small remarks

File: src/styles/style.css (hunk 3 above)
Comment (VladVeklenko) [reply src/styles/style.css:28]: Can't remove it, because when I remove it, I failed tests, unfortunately

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,60 @@
...
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body class="card">
+    <div 
+      class="container"
+      data-qa="card"
+    >
```
Comment (jstmpelowycz) [inline src/index.html:14]:
```suggestion
    <div class="container" data-qa="card">
```

File: src/index.html (hunk 5 above)
Comment (jstmpelowycz) [reply src/index.html:22]: agree, but up to you

File: src/index.html (hunk 1 above)
Comment (jstmpelowycz) [reply src/index.html:28]: agree, let's fix it

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,60 @@
...
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="container__price">
+        <p class="container__price-priceTag">
+          Price:
+        </p>
+
+        <p class="container__price-price">
```
Comment (jstmpelowycz) [inline src/index.html:46]:
```suggestion
      <div class="container__price">
        <p class="container__price-label">
          Price:
        </p>

        <p class="container__price-value">
```

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,60 @@
...
+          $2,199
+        </p>
+      </div>
+      <a
+      href="#Buy"
+      class="container__button"
+      data-qa="hover"
+    >
+      buy
+      </a>
```
Comment (jstmpelowycz) [inline src/index.html:56]:
```suggestion
      <a
        href="#Buy"
        class="container__button"
        data-qa="hover"
      >
        buy
      </a>
```

File: src/styles/style.css [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+  padding: 0 16px;
+}
+
+.container__img {
+  align-self: center;
+  margin-top: 32px;
+  min-height: 134px;
+  max-width: 160px;
+  margin-bottom: 40px;
+}
```
Comment (jstmpelowycz) [inline src/styles/style.css:25]:
```suggestion
.container__img {
  width: 160px;
  height: 134px;
  margin-top: 32px auto 40px;
}
```

File: src/styles/style.css [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+.container__description {
+  margin-bottom: 4px;
+  color: #060b35;
```
Comment (jstmpelowycz) [inline src/styles/style.css:29]: let's use variables, don't forget to fix it everywhere

File: src/styles/style.css [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (jstmpelowycz) [inline src/styles/style.css:87]: I don't think you need it, flexbox is for other purposes

File: src/styles/style.css [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+
+  height: 40px;
```
Comment (jstmpelowycz) [inline src/styles/style.css:89]: the checklist, point 8

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
almost done, let's take care of your code, see the comments 👇

File: src/styles/style.css (hunk 9 above)
Comment (VladVeklenko) [reply src/styles/style.css:25]: I set this margin and my code didn't  work properly

File: src/styles/style.css (hunk 12 above)
Comment (VladVeklenko) [reply src/styles/style.css:89]: In today's lessons, Paul also used height: 40 px, how I can set proper height without hardcore it?

Comment (jstmpelowycz) [reply src/styles/style.css:86]: line-height + paddings 🙂

**Review by jstmpelowycz — APPROVED** [review]

**Pull request closed without merging**